platform/darwin/src/MGLOfflineStorage.mm

```diff
@@ -153,72 +154,6 @@ - (void)setDelegate:(id<MGLOfflineStorageDelegate>)newValue {
     }
 }
 
-/**
- Returns the file URL to the offline cache, with the option to omit the private
- subdirectory for legacy (v3.2.0 - v3.2.3) migration purposes.
-
- The cache is located in a directory specific to the application, so that packs
- downloaded by other applications don’t count toward this application’s limits.
-
- The cache is located at:
- ~/Library/Application Support/tld.app.bundle.id/.mapbox/cache.db
-
- The subdirectory-less cache was located at:
- ~/Library/Application Support/tld.app.bundle.id/cache.db
- */
-+ (NSURL *)cacheURLIncludingSubdirectory:(BOOL)useSubdirectory {
```

Comment:
Previously these class methods were sandwiched below certain methods related to `+sharedOfflineStorage` and above `-init`. I’ve consolidated them down where the other database-related methods are implemented and renamed them for consistency, but otherwise I haven’t changed them.